### PR TITLE
Content

### PR DESCRIPTION
**Pull request by nlivni**
more updated content

### CODE REVIEW

File: choices.css [hunk 1]
```diff
@@ -68,8 +68,6 @@ footer{
     max-height:281px ;
     min-width:22%;
     margin:auto;
-    margin-left: 1em;
-    margin-right: 1em;
```
Comment (Unpaid-Intern) [inline choices.css]: Did you not care for the spacing?

File: templates/activities.js [hunk 2]
```diff
@@ -64,17 +77,19 @@ function monster_dance(person) {
 /*
     Using the monster to kill can act as garbage collection for unused people. It should have unpredictable consequences.
 */
-
 activity_deck.push(new Activity('kill', 'Play a little game', 'Your monster friend has an evil look in his eyes', 'Your monster friend has an evil look in his eyes', 0));
 function kill(person) {
-    player.updateHealth(1);
-    return "PLACEHOLDER TEXT FOR MONSTER KILL";
+    shuffle(person_deck);
```
Comment (Unpaid-Intern) [inline templates/activities.js]: do you actually pull this person out of their deck? I am unable to look at the code for shuffle(), so perhaps it is clearer int here.

File: templates/activities.js [hunk 3]
```diff
@@ -97,18 +115,16 @@ function baby_talk(person) {
...
         case 1:
             output = '<p>' + person.name + ' pay' + person.plural() + ' little attention to you. Turns out that ' +
             'weak parental relationships affect your health and happiness. Your speech also suffers.';
             player.updateHealth(-3);
-            player.happiness -= 3;
-            output += '<p>Health: ' + getSignedNumber(-3) + '<br>' + 'Happiness: ' + getSignedNumber(-3) + '</p>';
-            player.obituary[current_stage_id].push('You were born to parents who ignored you.');
+            player.updateHappiness(-3);
+            player.updateObituary('Ignored as a child.');
             break;
```
Comment (Unpaid-Intern) [inline templates/activities.js]: nice :)

File: templates/activities.js [hunk 4]
```diff
@@ -312,6 +366,106 @@ function set_social_class() {
...
+            output += 'You have fun with '+ person.name +' You are best friends!.';
+            break;
+        default:
+            player.updateHappiness(1);
+            output += 'Playing is fun. You pass the time while the grown ups drink.';
+            break;
+    }
+    return output;
+}
+
```
Comment (Unpaid-Intern) [inline templates/activities.js]: Awesome!  That all looks great

File: templates/people.js [hunk 5]
```diff
@@ -171,42 +172,94 @@ Person.prototype.plural = function() {
...
+ * @param {object} activities
+ * @param {number} connection
+ * @param {number} happiness
+ * @param {string} state
+ * @param {string} identity
+ * @param {number} stage
+ * @returns {Person}
+ */
+
+function createPerson(id, name, full_name, gender, met, activities, connection, happiness, state, identity, stage) {
```
Comment (Unpaid-Intern) [inline templates/people.js]: Cool.  I like this approach

File: choices.css (hunk 1 above)
Comment (nlivni) [inline choices.css]: I couldn't see the cards on a single line so I had to remove it. I kept
having to scroll.

File: templates/activities.js (hunk 2 above)
Comment (nlivni) [inline templates/activities.js]: not yet. just a placeholder for now but it'd be a good feature.

File: templates/people.js (hunk 5 above)
Comment (nlivni) [inline templates/people.js]: It's still very much a work in progress but I'm learning :)

File: templates/activities.js (hunk 3 above)
Comment (nlivni) [inline templates/activities.js]: yeah, it's getting a lot more writable. i need to write a few tests to help suss out where we'll have issues but the content is coming along!

**Comment by nlivni** [comment]
strongly suggest re-merging with development after this.

**Comment by Unpaid-Intern** [comment]
will do tonight.

**Pull request merged**